.github/workflows/linux-cpu-x64-build.yml

```diff
@@ -29,23 +32,12 @@ jobs:
         run: |
           mv ${{ env.ort_dir }} ort
 
-      - name: Git Submodule Update
-        run: |
-          git submodule update --init --recursive
-
       - name: Build with CMake and GCC
         run: |
           set -e -x
           rm -rf build
-          cmake --preset linux_gcc_cpu_release
-          cmake --build --preset linux_gcc_cpu_release
-          
-      - name: Build with CMake and clang
-        run: |
-          set -e -x
-          rm -rf build
-          cmake --preset linux_clang_cpu_release
-          cmake --build --preset linux_clang_cpu_release
+          cmake --preset linux_${{ matrix.compiler }}_cpu_release
+          cmake --build --preset linux_${{ matrix.compiler }}_cpu_release
```

Comment:
Why the arm one use docker but this one doesn't?

Reply:
It is fast to run natively

Reply:
If you prefer, we make them all run with Manylinux after pre-release, consistent with packingng pipeline. This way, we might be able to reuse the artifact, and consolidate Dockerfiles.